doc/langref.html.in

```diff
@@ -4635,6 +4635,14 @@ test "while null capture" {
     } else {
         try expect(sum2 == 3);
     }
+
```

Comment:
Add a short comment here to highlight what's going on?  Something like:

    // null capture with a continue expression 

(Alternatively, you could put this comment in the name of a new test block (similar to "while loop continue expression, more complicated" above)